api/api.py

```diff
@@ -120,11 +120,54 @@ def get_route_stops_with_connections(route_id):
     return route_stops
 
 
+def get_stop(stop_id):
+    result = db.engine.execute(
+        text(
+            """
+            SELECT 
+                osm_type,
+                osm_id,
+                name,
+                local_ref,
+                has_shelter,
+                has_bench,
+                has_tactile_paving,
+                has_departures_board,
+                is_wheelchair_ok,
+                st_asGeoJSON(ST_Transform(geom, 4326)) as geojson,
+                routes_at_stop
+            FROM d_stops
+            WHERE osm_id = :stop_id
+        """
+        ),
+        stop_id=stop_id,
+    )
+    try:
+        return dict(result.next().items())
+    except:
+        return False
+
+
 class Index(Resource):
     def get(self):
         return {"links": [{"href": "api.py/route/<route_id>"}]}
 
 
+osm_types = {0: "node", 1: "way", 2: "relation"}
```

Comment:
There is an inconsistency in the endpoints:
In the `routes/route_osm_id` response, we get `stop_list[].osm_id` with the following format: `osm_type/osm_id` (for example "node/5257992450")

So we should expect the `stop_osm_id` to keep that format in the `stops` endpoint too (and we expect this to work `http://localhost:5000/stops/node/5257992450`)

But I think we actually need to fix the `routes` endpoint (so it's quite unrelated to these changes and it can be done in another PR)

Reply:
It's a bad idea in rest api to use "/" in id. I suggest to change the stop is format and use standard identifier chars in it.

Reply:
thanks for the comments. I've changed the URLs : 
- http://localhost:5000/stops/5257205236 : will assume the id is a node
- http://localhost:5000/stops/5257205236?osm_type=way : will search for a way